metisp/workflows/metis/metis_classification.py

```diff
@@ -0,0 +1,42 @@
+from edps import classification_rule
+from . import metis_keywords as metis_kwd
+
+# Detector linearity calibration classification
+detlin_class = classification_rule("DETLIN_2RG_RAW",
```

Comment:
These classifications now all should have `lm` in their name, because they are not in an lm-specific file anymore. That is, how should we call the `DETLIN_GEO_RAW` now?

I think it would be easiest to just reuse the tag name, e.g. `detlin_2rg_raw`.